tldraw/tldraw
# [feature] `check-box` geo shape

This PR adds a `check-box` geo shape. 

![Kapture 2023-05-08 at 15 31 49](https://user-images.githubusercontent.com/23072548/236853749-99ba786f-73a4-4b65-86ca-f2cdac61a903.gif)

It also improves some logic around the `onClick` shape util handler and some surprisingly related fixes to point hit testing.

### Test Plan

1. Create a geo shape
2. Set it as a checkbox style
3. *easter egg* double click while holding alt to toggle between check-box and rectangle


### Release Note

- Adds checkbox geo shape.
